model/registry/process.yaml

```
@@ -41,6 +41,18 @@ groups:
           The PID of the process's group leader. This is also the process group
           ID (PGID) of the process.
         examples: [23]
+      - id: process.executable.build_id.gnu
```

```suggestion
      - id: process.executable.gnu.build_id
```

Would it make more sense to use the gnu namespace first, so other ELF gnu attributes can be added later?
Potentially `process.gnu.hash`, `process.gnu.version` could be added. (If changing to an ELF schema isn't done).

---

The [`.gnu.version`](https://refspecs.linuxbase.org/LSB_3.1.1/LSB-Core-generic/LSB-Core-generic/symversion.html#SYMVERTBL) section of an ELF shall contain the Symbol Version Table. It is not used to uniquely identify an ELF, like `.note.gnu.build-id` - which is proposed with this change.

Adding attributes for all section headers is not the intended purpose of this proposed change.

---

Switched namespaces as suggested with https://github.com/open-telemetry/semantic-conventions/pull/1329/commits/d779b0d81f1354f230bcd93b67b52f32dee6cfa0.

---

> The [`.gnu.version`](https://refspecs.linuxbase.org/LSB_3.1.1/LSB-Core-generic/LSB-Core-generic/symversion.html#SYMVERTBL) section of an ELF shall contain the Symbol Version Table. It is not used to uniquely identify an ELF, like `.note.gnu.build-id` - which is proposed with this change.
> 
> Adding attributes for all section headers is not the intended purpose of this proposed change.

Understood, I was just suggesting that to support these fields in the future, no need to add them now